htdocs/core/menus/standard/eldy.lib.php

```diff
@@ -87,7 +87,7 @@ function print_eldy_menu($db, $atarget, $type_user, &$tabMenu, &$menu, $noout =
 		'id' => $id,
 		'idsel' => 'home',
 		'classname' =>  $classname = (!empty($_SESSION["mainmenu"]) && $_SESSION["mainmenu"] == "home") ? 'class="tmenusel"' : 'class="tmenu"',
-		'prefix' => '<span class="fa fa-home fa-fw paddingright"></span>',
+		'prefix' => '<span class="fa fa-home fa-fw "></span>',
```

Comment:
Why removing paddingright ?

Reply:
Yes, I add new scss folder for isolate "section/layout/objects" styles. 
I create the scss folder for test new structure and new object segmentation (wip:)
_In this case, the icon is a part of the object named "list-item", at least that is the future intention..._
> what do you think ?
> I didn't ask you, is there a working folder to compile eldy style ?